noggin/templates/user-settings-otp.html

```diff
@@ -79,7 +79,7 @@ <h5 id="pageheading">OTP Tokens</h5>
   {% if otp_uri %}
   <script src="{{ url_for('static', filename='js/vendor/jquery-qrcode-1.0/jquery.qrcode.min.js') }}"></script>
   <script>
-    $(window).on('load',function(){
+    $(document).ready(function() {
```

Comment:
According to [the jQuery docs](https://api.jquery.com/ready/) you can also write `$(function() {...})`. Not sure which is the clearest, I was just mentioning it.